x-pack/plugins/security_solution/scripts/extract_tactics_techniques_mitre.js

```diff
@@ -11,12 +11,16 @@ const fs = require('fs');
 // eslint-disable-next-line import/no-extraneous-dependencies
 const fetch = require('node-fetch');
 // eslint-disable-next-line import/no-extraneous-dependencies
-const { camelCase, startCase } = require('lodash');
+const { camelCase, sortBy } = require('lodash');
 const { resolve } = require('path');
 
 const OUTPUT_DIRECTORY = resolve('public', 'detections', 'mitre');
-const MITRE_ENTERPRISE_ATTACK_URL =
-  'https://raw.githubusercontent.com/mitre/cti/master/enterprise-attack/enterprise-attack.json';
+
+// Every release we should update the version of MITRE ATT&CK content and regenerate the model in our code.
+// This version must correspond to the one used for prebuilt rules in https://github.com/elastic/detection-rules.
+// This version is basically a tag on https://github.com/mitre/cti/tags, or can be a branch name like `master`.
```

Comment:
Thanks for clarifying that 👍

Reply:
Yeah, sure! Don't think this is enough though. Ideally, we need to make this responsibility visible in our GH board and checklists and have a recurring ticket for it or try to automate it somehow as suggested in https://github.com/elastic/kibana/issues/89876.